auto_dev/data/templates/protocols/protodantic.jinja

```diff
@@ -13,9 +15,17 @@ from {{ primitives_import_path }} import (
     {%- endfor %}
 )
 
+# ruff: noqa: N806, C901, PLR0912, PLR0914, PLR0915, A001
+# N806     - variable should be lowercase
+# C901     - function is too complex
+# PLR0912  - too many branches
+# PLR0914  - too many local variables
+# PLR0915  - too many statements
+# A001     - shadowing builtin names like `id` and `type`
+
```

Comment:
auto generated code is simply prone to this, hence we ignore them. N806 is a specific little wacky hack i had to perform for `protodantic.py` output to be compatible with AEA framework (without making this module dependent on AEA quirks)